Whoaverse/Whoaverse/Scripts/whoaverse.js

```diff
@@ -204,7 +204,119 @@ $(document).ready(function () {
             }
         });
     });
-
+    
+    var key = {};
+
+    // List of the inputs the user use when creating a new discussion
+    var submitInputs = {
+      post: [
+        '#Title',
+        '#Subverse',
+        '#MessageContent'
+      ],
+      reply: [
+        '#CommentContent'
+      ]
+    };
+    
+    document.onkeydown = function(evt) {
+      var evt = evt || window.event;
+      var charCode = evt.keyCode || evt.which;
+      var charStr = String.fromCharCode(charCode);
+      
+      // Turn key numbers into readable string
+      switch(charCode) {
```

Comment:
Why are we handling all these other keys when only 2 keys are relevant for this PR: charcode 17 and 13?